docs/tr/index.md

```diff
@@ -235,19 +236,78 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 
 
 
+## Ekip
```

Comment:
I think the issue is with the table above. There is something that breaks all the table down.
Have you tried running the website locally?